kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternFieldsController.java

```diff
@@ -114,7 +116,8 @@ private void initialize() {
         displayNameTextField.setEditable(false);
         patternFieldsViewModel.getProperty(MEANING_ENTITY).subscribe(meaningObject -> {
             if (meaningObject != null) {
-                displayNameTextField.setText(((ConceptEntity)meaningObject).description());
+                ConceptEntity conceptEntity = Entity.getFast((EntityFacade) meaningObject);
```

Comment:
Can we use the ViewProperties calculator?
```java
viewProperties.calculator().getFullyQualifiedDescription(nid).ifPresentOrElse();
or
viewProperties.calculator().getRegularDescriptionText(nid).ifPresentOrElse();
or
String descripText = viewProperties.calculator().getDescriptionText(nid).get()
```

Reply:
we shouldn't really use the convenience method `.description()` due to concepts from starter data that could be part of bindings (generated code) or may not give you the latest entity version. 

View Calculators are quick queries (cache) that pull from the database. UI developers really should be using view calculators for displays and Entity / db services for transactions.

Reply:
@carldea 
Good catch.
However that was already there before this PR, that is: displayNameTextField.setText(((ConceptEntity)meaningObject).description()).
Want to make a new one to fix those view calculator not being called issues?